.github/workflows/test_rsync_test_script.yml

```diff
@@ -0,0 +1,47 @@
+name: test rsync check
+
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      rose_ref:
+        description: The Rose branch to test against
+        required: true
+  schedule:
+  - cron: '37 04 * * 1-5' # 03:37, Monday-Friday
```

Comment:
Having this both on `pull_request` and `schedule` is overkill.

How about running this on `pull_request` but only running it when either `metomi/rose/loc_handlers/rsync_remote_check.py` or this test file are modified.